torch/onnx/symbolic_opset11.py

```diff
@@ -11,13 +11,28 @@
 # This file exports ONNX ops for opset 11
 
 black_listed_operators = [
-    "eq", "ne", "scatter", "clamp", "clamp_min", "clamp_max", "sort", "topk", "hardtanh"
```

Comment:
Do we need to add clamp_min/max?

If not, did we accidentally put them in the black list?

Reply:
We don't need to, they are helper functions called only by 'clamp' itself.